scripts/version-sync.sh

```diff
@@ -111,6 +111,7 @@ MAIN_VERSION=$(git show origin/main:unstructured/__version__.py | grep -o -m 1 -
 MAIN_IS_RELEASE=false
 [[ $MAIN_VERSION != *"-dev"* ]] && MAIN_IS_RELEASE=true
 CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+MAIN_IS_RELEASE=true
```

Comment:
this testing artifact should be removed, other than that version-sync changes have been tested and LGTM

Reply:
Good catch, removed in [2a92e96](https://github.com/Unstructured-IO/unstructured/pull/1835/commits/2a92e96182ac7ce9ef63d9c4cf5e4b1639eaf47a).